region/norfair/west/Ice Beam Tutorial Room.json

```diff
@@ -360,6 +409,79 @@
       },
       "bypassesDoorShell": true
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Morph, Damageless",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        {"or": [
+          {"and": [
+            "h_canArtificialMorphMovement",
+            "h_lavaProof"
+          ]},
+          {"and": [
+            "Morph",
+            "canTrickyUseFrozenEnemies"
+          ]},
+          {"and": [
+            "h_canArtificialMorphIBJ",
+            "h_canArtificialMorphBombHorizontally",
+            "canTrickyUseFrozenEnemies"
+          ]},
+          {"and": [
+            "h_canArtificialMorphHBJ",
```

Comment:
I did this with an IBJ+bomb horizontally on the right, and a bomb horizontally into unmorph+down back on the left.  I couldn't make the jump with the bomb alone.  I had rolling speed when getting hit by the bomb.

The left side HBJ looks like you sit on the farthest pixel over and just time the HBJ right...  It's harder than a regular HBJ.  You can also start an IBJ facing forward or backwards with the same setup and horizontal boost from there.  Or DIBJ.

I don't like the consistency of any of these options.

Reply:
We don't yet have a tech for rolling speed + bomb horizontally. I think there would be quite a few places it could be used, but it feels rough.

I added a dev note and the dibj option. Do you think it should be changed further?